elastic/kibana
# [Infrastructure UI] Fix table position shifting

closes [#160268](https://github.com/elastic/kibana/issues/160268)
## Summary

Fixes table moving down when the in-table filter is displayed

https://github.com/elastic/kibana/assets/2767137/ff781aa4-53c9-44fa-9915-4da03a66f672





### Bonus
Added transition when the button becomes visible:

https://github.com/elastic/kibana/assets/2767137/57e17a9a-8606-431f-9710-5d839078fbe8

### How to test
- Start a local Kibana instance
- Navigate to `Infrastructure > Hosts`
- On the table, interact with the check boxes.
